### DISCUSSION TITLE
Execute the command immediately after picking it from history search

### DISCUSSION THREAD

**Discussion by zeionara** [marlonrichert/zsh-autocomplete#653] [Q&A] [answered]
Hi, I wonder is it possible to execute the first command that I choose from history search when I press `Enter`, so that the shell instead of appending `;` and waiting until I pick the next command immediately executes the first one? I rarely choose more than one command from history and just press `Enter` two times in a row.

I would like this to work only for history search - for example, for paths autocompletion everything must work as before.

**Comment by marlonrichert** [comment, marked as answer]
This already [documented in the Readme](https://github.com/marlonrichert/zsh-autocomplete#make-enter-submit-the-command-line-straight-from-the-menu).
